server/src/controllers/authTeamController.js

```diff
@@ -0,0 +1,102 @@
+import rp from 'request-promise';
+
+const server = 'http://localhost:8080';
```

Comment:
Agreed, I think we can let this go for now and it can something we refactor, I'll make an issue for it